madara/crates/primitives/oracle/src/pragma.rs

```diff
@@ -52,29 +52,29 @@ impl PragmaOracle {
 
 #[async_trait]
 impl Oracle for PragmaOracle {
-    /// Methods to retrieve ETH/STRK price from Pragma Oracle
+    /// Methods to retrieve STRK/ETH price from Pragma Oracle
     ///
     /// Return values:
     /// Ok((u128, u32)) : return the price tuple as (price, decimals)
-    /// Err(e) : return an error if anything went wrong in the fetching process or eth/strk price is 0
-    async fn fetch_eth_strk_price(&self) -> anyhow::Result<(u128, u32)> {
+    /// Err(e) : return an error if anything went wrong in the fetching process or STRK/ETH price is 0
```

Comment:
if strk/eth price is 0 then we don't have a job anymore, makes sense to return an error i guess